apps/bors_frontend/web/views/layout_view.ex

```diff
@@ -5,4 +5,43 @@ defmodule BorsNG.LayoutView do
   """
 
   use BorsNG.Web, :view
+
+  def get_version do
+    get_heroku_commit() || get_git_commit() || get_release_version()
+  end
+
+  def get_heroku_commit do
+    "HEROKU_SLUG_COMMIT"
+    |> System.get_env()
+    |> case do
+      nil -> nil
+      commit -> String.slice(commit, 0..10)
+    end
+  end
+
+  def get_git_commit do
+    ".git/HEAD"
+    |> File.read()
+    |> case do
+      {:ok, "ref: " <> branch} ->
+        branch = String.trim(branch)
+        case File.read(".git/" <> branch) do
+          {:ok, commit} -> String.trim(commit)
+          _ -> nil
+        end
+      {:ok, commit} -> String.trim(commit)
+      _ -> nil
+    end
+    |> case do
+      nil -> nil
+      commit -> String.slice(commit, 0..10)
+    end
+  end
+
+  def get_release_version do
+    case :application.get_key(:vsn) do
```

Comment:
vsn is the version provided in the Mix.exs file. It's the fallback in case we can't find a git commit anywhere else.